features/bootstrap/FeatureContext.php

```diff
@@ -355,6 +421,66 @@ public function thereIsDummyObjectsWithDummyBoolean($nb, $bool)
     }
 
     /**
+     * @Given there is :nb embedded dummy objects with embeddedDummy.dummyBoolean :bool
+     */
+    public function thereIsDummyObjectsWithEmbeddedDummyBoolean($nb, $bool)
+    {
+        if (in_array($bool, ['true', '1', 1], true)) {
```

Comment:
This wasn't part of this PR or was it? I think there was a reason we didn't use that. I'll look into this.